### ISSUE TITLE
Paper-textarea label not moving as a floating label  

### ISSUE THREAD

**Issue by lpellegr** [PolymerElements/paper-input#555]
### Description

When a `paper-textarea` is empty and a call is done to set a new value programmatically, the new content overlaps the label. 

Initial rendering:
![screenshot from 2017-07-08 18-51-54](https://user-images.githubusercontent.com/128898/27987478-9fe7f7c0-640e-11e7-94f0-945e8134c90a.png)

After the value is set programmatically:

![screenshot from 2017-07-08 19-06-27](https://user-images.githubusercontent.com/128898/27987607-9ea9abb8-6410-11e7-9a52-26ebf99ddd12.png)

### Expected outcome

I expect the label to move as a floating label and not stay a resting label:

![screenshot from 2017-07-08 19-06-42](https://user-images.githubusercontent.com/128898/27987609-a340341c-6410-11e7-9f25-e0cbfbec9d7d.png)

### Actual outcome

The floating label does not move as a floating label.

### Steps to reproduce

0. Uses Polymer 2
1. Add a `paper-textarea` to your page:
```
<paper-textarea class="form-group" id="content" label="Content" max-rows="0" name="postContent" rows="5" required></paper-textarea>
```
2. In `ready()` method or on button tap, set the value programatically:
```
this.$.content.value = 'test';
```

I have tried to use `updateValueAndPreserveCaret` but the issue is the same.

The issue occurs only when the initial content is empty.

**Comment by Lange** [comment]
I am having this issue as well.

**Comment by denishamann** [comment]
Hello, I also have the same issue since a couple of days and can't find the reason why.

**Comment by notwaldorf** [comment]
If it *just* broke, would you mind updating web components and trying again? There was a polyfill problem that I think may be the culprit, since we haven't pushed any new changes to this element recently

**Comment by krumbunny** [comment]
It's been broken for awhile, at least since February

**Comment by yairopro** [comment]
Hello. 
I've got the same problem.
Hope this problem will soon be solved.
Thanks a lot @krumbunny for the workaround. It works like a charm. I just have to bind the `paper-input.value` property to this `always-float-label` property to fix the problem.

**Comment by shaeberling** [comment]
I just ran into the same issue today. Any ETA on a fix? Seems like a pretty common problem.

**Comment by Ravikumar-Dhanasekaran** [comment]
Similar issue. Not able to fix. Please post any workaround.?

**Comment by yairopro** [comment]
Here's a workaround:
`<paper-textarea always-float-label="[[value]]" value="{{value}}"></paper-textarea>`
This way it forces the label to float if there's a value in the input.

**Comment by kreinholt** [comment]
I believe this is caused by the following commit: https://github.com/PolymerElements/iron-autogrow-textarea/commit/2dba77f1e960c472a3bf9353ffab5a3fb114228d. The line `this.fire('bind-value-changed', {value: this.bindValue});` has been commented out.

**Comment by catiejojo** [comment]
Thank you, @kreinholt! I was also struggling with this bug--I tried uncommenting it and the functionality went back to normal. Also props to the programmer that put this comment above the commented code it: `This code is from early 1.0, when this element was a type extension. It's unclear if it's still needed, but leaving in in case it is.` It's far from best practice to edit the code directly like this, but I think it's the easiest workaround until there's a less hacky/1.0 fix.

**Comment by yglodt** [comment]
I see the same problem on a `paper-input`.

**Comment by rbjarnason** [comment]
I see the same problem when porting my app from 1.x to 2.x. What is the current recommend workaround? I'd rather not go to a fork.

**Comment by notwaldorf** [comment]
Marking this as a duplicate of https://github.com/PolymerElements/paper-input/issues/508 (and looking at that PR which should fix both of these right now)

**Comment by notwaldorf** [comment]
(fixed with the 2.1.1 release of iron-autogrow-textarea btw)
